src/rspack/utils.ts

```diff
@@ -20,11 +20,9 @@ export class FakeVirtualModulesPlugin {
   static counters: Map<string, number> = new Map<string, number>()
 
   static {
-    ['SIGINT', 'SIGTERM', 'SIGQUIT', 'exit'].forEach((event) => {
-      process.once(event, () => {
-        this.counters.forEach((_, dir) => {
-          fs.rmSync(dir, { recursive: true, force: true })
-        })
+    process.once('exit', () => {
```

Comment:
Can we move this part to the constructor? The static block will always be executed, regardless of whether rspack is used.

Reply:
Sure, is something like [this](https://github.com/unjs/unplugin/pull/541/commits/c48bb90a9e1df7da0921e0d87fa65c68c16af2f1) okay?